packages/shared/src/components/profile/devcard/DevCardStats.tsx

```diff
@@ -1,24 +1,26 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import { EyeIcon, ReputationIcon } from '../../icons';
+import { EyeIcon, ReadingStreakIcon, ReputationIcon } from '../../icons';
 import { DevCardStatsSection } from './DevCardStatsSection';
 import { PublicProfile } from '../../../lib/user';
 
 interface DevCardStatsProps {
   user: PublicProfile;
   className?: string;
   articlesRead: number;
+  longestStreak: number;
 }
 
 export function DevCardStats({
   user,
   className,
   articlesRead,
+  longestStreak,
 }: DevCardStatsProps): ReactElement {
   return (
     <span
       className={classNames(
-        'flex w-full flex-row gap-3 rounded-16 bg-raw-pepper-90 p-4 shadow-2',
+        'flex w-full flex-row gap-3 rounded-16 bg-raw-pepper-90 px-4 py-2 shadow-2',
```

Comment:
Just checking this should be pepper always?

Reply:
I believe yes, their background doesn't changed based on the theme